### PR TITLE
safe check for escape characters

### PR DESCRIPTION
**Pull request by abhi580**
Added the safe check for passwords. This is tested for all the special characters.

### CODE REVIEW

File: src/appliance-onboarding-script/appliance_setup/pkgs/_utils.py [hunk 1]
```diff
@@ -107,14 +107,15 @@ def create_dir_if_doesnot_exist(path: str):
     except FileExistsError:
         pass
 
-def quote_string_for_cmd(ip: str):
-    return f'"{ip}"'
+def safe_escape_characters(ip:str):
+    safe_str = ''
+    for ch in ip:
+        if ch == '"':
+            safe_str = safe_str + '\\' + ch
```
Comment (uthsab) [inline src/appliance-onboarding-script/appliance_setup/pkgs/_utils.py:114]: >'\\' + ch [](http://example.com/codeflow?start=34&length=9)

Will this escaping work in both windows and linux environment?

**Review by uthsab — CHANGES_REQUESTED** [review]
:clock1:

File: src/appliance-onboarding-script/appliance_setup/pkgs/_utils.py (hunk 1 above)
Comment (abhi580) [reply src/appliance-onboarding-script/appliance_setup/pkgs/_utils.py:114]: tested in linux and windows environment. It is working fine.

**Review by uthsab — APPROVED** [review]
:shipit:

**Pull request merged**